hack/invalid-examples/httproute/invalid-path.yaml

```diff
@@ -0,0 +1,10 @@
+kind: HTTPRoute
+apiVersion: gateway.networking.k8s.io/v1alpha2
+metadata:
+  name: invalid-path
+spec:
+  rules:
+  - matches:
+    - path:
+        type: Prefix
+        value: /<>invalid
```

Comment:
Why is this invalid? `<>` is valid in a url

Reply:
Wouldn't this be an invalid path segment per the RFC? Regardless, this was a good reminder that my validation was too strict here simply because we allow RegExp and ImplementationSpecific matching here. I've removed this.